src/DetailsView/components/details-view-command-bar.tsx

```diff
@@ -199,6 +206,16 @@ export class DetailsViewCommandBar extends React.Component<
         });
     }
 
+    private renderSaveAssessmentButton = (): JSX.Element | null => {
```

Comment:
It looks like the Save Assessment button is showing up in FastPass as well as in Assessment; is this expected at this point? 
![image](https://user-images.githubusercontent.com/1752950/96659473-ddab1480-12fb-11eb-86d2-831cc4a0215f.png)

I was going to observe that `renderSaveAssessmentButton` never returns `null`, but maybe it wants to for FastPass? It may help to look at the logic for the Start Over button/menu which has different behavior for FastPass and Assessment (see also details-view-switcher-nav.ts). Given this is all behind a private feature flag, it might be OK to defer only showing for assessments to a follow-up PR.

Reply:
Thank you, this is helpful. Will introduce this in a separate PR, per our discussion, thank you!